docs/source/api/logprob.rst

```diff
@@ -21,3 +21,51 @@ Conditional probability
 
    conditional_logp
    transformed_conditional_logp
+
+
+Derived probability functions
+-----------------------------
+
+In PyMC users can derive their own custom distributions. Custom distribution refers to the ability to define and use probability distributions that are not included in the standard set of distributions provided.
+While PyMC provides a wide range of common probability distributions (e.g., Normal, Bernoulli, etc.), there may be cases where you need to use a distribution that is not available by default. In such cases, you can create your own custom distribution using the pm.CustomDist class provided by PyMC.
+Simplest way to define a Custom Distribution can be better understood from the following example:
```

Comment:
```suggestion
The use of the :class:`~pm.CustomDist` API can be better understood from the following example:
```
There are two ideas combined into one that is not intuitively phrased. Either go "The simplest way to define a custom distribution in PyMC is..." or "Using pm.CustomDist can be illustrated with...". I prefer the latter